jina/drivers/control.py

```diff
@@ -158,3 +160,46 @@ def __call__(self, *args, **kwargs):
 
 class ForwardDriver(RouteDriver):
     """Alias to :class:`RouteDriver`"""
+
+
+class PngToDiskDriver(FlatRecursiveMixin, BaseRecursiveDriver):
+    """A driver that can store an intermediate represenation of a png in the workspace, under a given folder.
+
+    Useful for debugging Crafters in the Flow
+    """
+
+    def __init__(self, workspace=None, prefix='', top=10, *args, **kwargs):
+        self.prefix = prefix
+        self.top = top
+        self.done = 0
+        self.workspace = workspace or os.getenv('JINA_WORKSPACE') or 'workspace'
+        self.folder = os.path.join(self.workspace, self.prefix)
+        if not os.path.exists(self.folder):
+            os.makedirs(self.folder)
+        super().__init__(*args, **kwargs)
+
+    def _apply_all(
+        self,
+        docs: 'DocumentSet',
+        *args,
+        **kwargs,
+    ) -> None:
+        def _move_channel_axis(
+            img: 'np.ndarray', channel_axis_to_move: int, target_channel_axis: int = -1
+        ) -> 'np.ndarray':
+            if channel_axis_to_move == target_channel_axis:
+                return img
+            return np.moveaxis(img, channel_axis_to_move, target_channel_axis)
+
+        def _load_image(blob: 'np.ndarray', channel_axis: int):
+            from PIL import Image
```

Comment:
as an external import, it should be imported with the help of `ImportChecker`

Reply:
I can't find any usages of this class

It's added here but this method is never called

def check(args: 'Namespace'):
    """Check jina config, settings, imports, network etc"""
    from jina.checker import ImportChecker

    ImportChecker(args)